backup/setup/courses.json

```diff
@@ -66,7 +67,7 @@
 			"eventId" : "457b4a47-c72a-4553-95e4-62351aed493c"
 		}
 	],
-	"color" : "#ffb9b9",
+	"color" : "#795548",
```

Comment:
Are these color changes done by any specific permutation? because the changes done are not listed in the ticket (same for the rest of the seed data).
Also the colors in the seed data for teams are not changed.

Reply:
> Are these color changes done by any specific permutation? because the changes done are not listed in the ticket (same for the rest of the seed data).
> Also the colors in the seed data for teams are not changed.

What Martin says should be actually done.

Reply:
no specific permutation, since the seeddata is actually not using the current course colors.
the teams thing is a good point